features/frameworks/mbed-trace/MbedTraceInitialiser.cpp

```diff
@@ -0,0 +1,12 @@
+#include "mbed_trace.h"
+
+class MbedTraceInitialiser {
```

Comment:
might inherit from noncopyable class? `: private mbed::NonCopyable`


Only purpose is initialization (thus the Initialiser class name) ? No future for providing C++ methods for tracing, C only? Just checking

Reply:
I’ve thinking that c++ APi top of C lib could give some benefits. In fact I did some c++ wrapper POC.. Anyway, maybe we should keep that an option in future..